tests/operator_test.go

```diff
@@ -981,18 +981,24 @@ var _ = Describe("ALL Operator tests", func() {
 					Skip("This test depends on prometheus infra being available")
 				}
 
-				numAddedStorageClasses = 2
 				defaultStorageClass := utils.GetDefaultStorageClass(f.K8sClient)
 				defaultStorageClassProfile := &cdiv1.StorageProfile{}
 				err := f.CrClient.Get(context.TODO(), types.NamespacedName{Name: defaultStorageClass.Name}, defaultStorageClassProfile)
 				Expect(err).ToNot(HaveOccurred())
+
+				originalMetricVal := 0
```

Comment:
I think originally the metric values is not 0 because we have the "local" storage class

Reply:
But local should not trigger the warning anymore since we added it to the storage profile.

Reply:
did we? where?

Reply:
Due to PR #2546 playing with CDI CR, in this test case it takes `StorageProfileReconciler` a second or two to get to its `IncompleteProfileGauge.Set(>0)`.

Reply:
Interesting we only did it for LSO not U/S local storage.

Reply:
yeah because you still need to alert when someone is rolling their own static PVs